rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -905,6 +916,58 @@ public void updateCollections(Transaction rskTx) throws IOException {
         updateFederationCreationBlockHeights();
     }
 
+    private void processFundsMigration(Transaction rskTx) throws IOException {
+        Wallet retiringFederationWallet = activations.isActive(RSKIP294) ?
+            getRetiringFederationWallet(true, bridgeConstants.getMaxInputsPerPegoutTransaction()) :
+            getRetiringFederationWallet(true);
+
+        List<UTXO> availableUTXOs = getRetiringFederationBtcUTXOs();
+        Federation activeFederation = getActiveFederation();
+
+        if (federationIsInMigrationAge(activeFederation) && hasMinimumFundsToMigrate(retiringFederationWallet)) {
+            logger.info(
+                "Active federation (age={}) is in migration age and retiring federation has funds to migrate: {}.",
+                rskExecutionBlock.getNumber() - activeFederation.getCreationBlockNumber(),
+                retiringFederationWallet.getBalance().toFriendlyString()
+            );
+
+            migrateFunds(
```

Comment:
Probably the other try catch shouldn't be there, since the only exception migrateFunds could throw is an IOException accessing the storage. In that case we want the exception to be caught by the Bridge.

We have a pending analysis to do in that part of the code anyway, it's possible it is not being executed at all

Reply:
Cool. Let's create a task for it.